lib/RoutingList/hooks/useRoutingList/useRoutingList.js

```diff
@@ -0,0 +1,54 @@
+import {
+  flatten,
+  uniq,
+  keyBy,
+} from 'lodash';
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import { LIMIT_MAX, ROUTING_LIST_API, USERS_API } from '../../../constants';
+import { batchRequest } from '../../../utils';
+
+const DEFAULT_DATA = [];
+
+function getUserName(user) {
+  return `${user.personal?.firstName} ${user.personal?.lastName}`;
```

Comment:
If user `firstName` does not exist you will have the following result `undefined lastName`.
Is it expected?